### DISCUSSION TITLE
How can I get the current relative $record from within the RepeatableEntry?

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#12814] [Help] [answered]
### Package

Infolist builder

### Package Version

v3

### How can we help you?

If I use state() on infolists's RepeatableEntry to get current record in it gives parent record, not the relative record.  How can I access the current record?  Thanks

**Comment by leandrocfe** [comment]
Should be `$this->getRecord()` or `$livewire->getRecord()`

**Reply by myregistration** [reply]
Those didn't work.  I'm using it in a Relation Manager, if that makes a difference.  I'm trying to access the current record of the RepeatableEntry so I can use it within the schema.  Thanks

**Reply by myregistration** [reply]
This did help me for another issue.  Thanks!

**Comment by myregistration** [comment, marked as answer]
Turns out I didn't need to access the iterated record.  I am trying to re-use form and infolist code by storing it on the model and couldn't get the current record to initialize it, but turns out I didn't need to.  Since the record is current I could just use `->schema(Pet::getInfoList())` without having to use something like` $record::getInfoList()` or `$record->pet::getInfoList()`.  I still think it would be helpful if we could access the current record within `RepeatableEntry`, I could only access the parent via `->schema(fn ($record) => dd($record))`.
